.travis.yml

```diff
@@ -59,10 +59,12 @@ install:
         sed -i.bak "s/plugins *= *Cython\.Coverage//g" $cf;
       done
     fi
+  # temporarily use older version of pyarrow
+  - pip install "pyarrow<0.14"
+  - conda install --quiet --yes -n test -c conda-forge python=$PYTHON "tiledb-py>=0.4" pyarrow || true
   - pip install -r requirements-dev.txt &&
     pip install -r requirements-extra.txt &&
     pip install virtualenv coveralls flake8 etcd-gevent
-  - conda install --quiet --yes -n test -c conda-forge python=$PYTHON "tiledb-py>=0.4" || true
   - virtualenv testenv && source testenv/bin/activate
```

Comment:
We need to enforce the pyarrow version in requirements.txt, rather than ci configuration. Otherwise, our users who use py36 will be affected after they upgrade mars.

Reply:
I think we can limit the ci version first, and then find a compatible way for new version of pyarrow. The limitation like ">=0.11,<0.14" is sometimes annoying especially packages conflict.

Reply:
The reason to add this is that we encounter an ImportError, not compatibility issue. This is only a workaround and I think pyarrow will fix that soon.